docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,160 @@
+
+* **Add hot key**: Add a new hot key that can be used to manage the neuron. This provides an alternative to using the principal’s cold key to manage the neuron, which might be onerous and difficult to keep secure, especially if it is used regularly. A hot key might be a WebAuthn key that is maintained inside a user device, such as a smartphone.
+
+* **Remove hot key**: Temove a hot key that has been previously assigned to the neuron.
+
+```
+type Configure = record { operation : opt Operation };
+type Operation = variant {
+  RemoveHotKey : RemoveHotKey;
+  AddHotKey : AddHotKey;
+  ...
+};
+type AddHotKey = record { new_hot_key : opt principal };
+type RemoveHotKey = record { hot_key_to_remove : opt principal };
+```
+
+### Voting with a neuron
+* **Follow relationships (mapping from topic to list of followers)**: A neuron can be configured to vote automatically by following other neurons on a topic-by-topic basis. For any valid topic, a list of followers can be specified, and the neuron will follow the vote of a majority of the followers on a proposal with a type belonging to that topic. If a null topic is specified, this acts as a catch-all that enables the neuron to follow the vote of followees where a rule has not been specified.
+
+* **Recent votes**: A record of recent votes is maintained. This can provide a guide for those wishing to evaluate whether to follow a neuron or how their followers are voting.
+
+* **Vote**: Have the neuron vote to either adopt or reject a proposal with a specified ID.
+* **Follow**: Add a rule that enables the neuron to vote automatically on proposals that belong to a specific topic, by specifying a group of followee neurons whose majority vote is followed. The configuration of such follow rules can be used to:
+    - Distribute control over voting power amongst multiple entities.
+    - Have a neuron vote automatically when its owner lacks time to evaluate newly submitted proposals.
+    - Have a neuron vote automatically when its own lacks the expertise to evaluate newly submitted proposals.
+    - For other purposes.
+
+A follow rule specifies a set of followers. Once a majority of the followers vote to adopt or reject a proposal belonging to the specified topic, the neuron votes the same way. If it becomes impossible for a majority of the followers to adopt (for example, because they are split 50–50 between adopt and reject), then the neuron votes to reject. If a rule is specified where the proposal topic is null, then it becomes a catch-all-follow rule, which will be used to vote automatically on proposals belonging to topics for which no specific rule has been specified. If the list of followers is empty, this effectively removes the following rule.
+
+TODO: HOW TO GET THE ID OF TOPICS BEFORE DOING THIS?
```

Comment:
@max-dfinity is this something that you can hep with? How do I learn form the outside which topic has which ID so that I can set following? I am assuming there must be an API for it?